ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/computation/CommandRunnerTest.java

```diff
@@ -59,16 +66,25 @@ private CommandRunner getCommanRunner(StatementExecutor statementExecutor) {
     return commandRunner;
   }
 
+  private StatementExecutor getStatementExecutor() {
+    Map<String, Object> props = new HashMap<>();
+    props.put("application.id", "ksqlStatementExecutorTest");
+    props.put("bootstrap.servers", CLUSTER.bootstrapServers());
 
-  @Test
-  public void testThread() throws InterruptedException {
     MockKsqkEngine mockKsqkEngine = new MockKsqkEngine(
-        TestUtils.getMockKsqlConfig(), new MockKafkaTopicClient());
+        new KsqlConfig(props), new MockKafkaTopicClient());
+
     StatementParser statementParser = new StatementParser(mockKsqkEngine);
-    StatementExecutor statementExecutor = new StatementExecutor(mockKsqkEngine, statementParser);
+
+    return new StatementExecutor(mockKsqkEngine, statementParser);
+  }
+
+  @Test
+  public void testThread() throws InterruptedException {
```

Comment:
What is this testing?

Reply:
Renamed the method for better description.